### DISCUSSION TITLE
[analog] multiply by 2 the analog value in volts

### DISCUSSION THREAD

**Discussion by lboue** [arendst/Tasmota#20359] [Q&A] [answered]
Hello,

I'm looking for a way to multiply by 2 the analog value in volts measured by a bridge divider.
Is there a way to do this with Tasmota?

![image](https://github.com/arendst/Tasmota/assets/938089/42142890-4e11-482b-ab2e-aca98b629987)

Regards

**Comment by sfromis** [comment, marked as answer]
If you config the analog input for `ADC Range`, you can use the [AdcParam](https://tasmota.github.io/docs/ADC/#commands) command to control a linear range remapping of values, easily encompassing a multiplication by 2.

**Reply by lboue** [reply]
It worked with this. Thanks
`AdcParam 6, 0, 1024, 0, 2080`
